lib/manifest.js

```diff
@@ -122,6 +122,17 @@ internals.manifest = [
         list: false,
         after: ['services']
     },
+    {
+        place: 'caches',
+        method: 'cache.provision',
+        async: true,
+        list: true,
+        useFilename: internals.passthruOn('name'),
+        example: {
+            engine: null,
+            name: ''
+        }
+    },
     {
         place: 'plugins',
         method: 'register',
```

Comment:
The purpose of the `after` property is to express exactly this sort of rule: "plugin registrations come _after_ cache provisions."  Can you also add `after: ['caches']` to this plugin registrations item?  It's described in a bit more detail [here](https://github.com/hapipal/haute-couture/blob/master/API.md#hautecoutureusingdirname-amendments), but I am happy to elaborate!